README.md

```diff
@@ -14,9 +14,9 @@ All the documentation is located in the `/docs` folder in its raw form. A more w
 
 New builds of OpenKH are automatically generated every time one of the contributors inspects and approves a new proposed feature or fix. Those builds are considered stable as they are built from the `master` branch. The version format used in the builds is `YEAR.MONTH.DAY.BUILDID`.
 
-[![OpenKh](https://img.shields.io/badge/OpenKh-Download-blue.svg)](https://github.com/Xeeynamo/OpenKh/releases)
+[![OpenKh](https://img.shields.io/badge/OpenKh-Download-blue.svg)](https://github.com/OpenKH/OpenKh/releases)
 
-All the builds from `master` and from pull requestes are generated from [Azure Pipelines](https://dev.azure.com/xeeynamo/OpenKH/_build).
+All the builds from `master` and from pull requestes are generated from [Azure Pipelines](https://dev.azure.com/OpenKH/OpenKH/_build).
```

Comment:
Based on PR 695 releases are no longer on azure pipelines and made using github actions. Also the link is invalid  because OpenKH does not have an azure account (not sure exactly how to word this). This statement just seems outdated and maybe should just be removed since its all done from github for builds unless im wrong